src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
```

Comment:
Consider using a constant for the daily rental cost instead of a literal to make the code more maintainable and readable. For example, `const DAILY_RENTAL_COST = 40;`.